back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -79,8 +79,9 @@ def create
   def copy
     source_project = Project.find(params[:id])
 
-    @project = LocalProjectCopyService.new.copy(source_project)
+    @project = Project.new
```

Comment:
I think it won't work with moderators because of this https://github.com/CitizenLabDotCo/citizenlab/blob/41a01fc0f84a2b032a0639558f38787a012cbaee/back/app/policies/project_policy.rb#L87

Maybe you can try this `@project = Project.new(folder: source_project.folder)`

Reply:
Not sure we want moderators to use this feature.

The [specs say](https://www.notion.so/citizenlab/Copy-Projects-4725ad84fa65452294d04dc90e28358c#e3deb9a0733343a799fbfc3fcbce1d18): "Only admins can copy the project."

But maybe I should check that "Admins" doesn't include Project Moderators.

Reply:
> Not sure we want moderators to use this feature.

If not, it's fine then.

Reply:
I've asked Simon F for clarity ([Slack thread](https://citizenlabco.slack.com/archives/C04HW4R2C3Y/p1674153640351339)) and also noted need to check this on [next BE ticket](https://citizenlab.atlassian.net/browse/CL-2620) for this epic.